### ISSUE TITLE
CT: Uncaught Error: Assertion failed: Accessibility is provided by RectangularRadioButton...

### ISSUE THREAD

**Issue by jbphet** [phetsims/isotopes-and-atomic-mass#108]
The following error is occurring on CT for this sim:

```
isotopes-and-atomic-mass : fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1656695400610/isotopes-and-atomic-mass/isotopes-and-atomic-mass_en.html?continuousTest=%7B%22test%22%3A%5B%22isotopes-and-atomic-mass%22%2C%22fuzz%22%2C%22unb:uilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1656695400610%22%2C%22timestamp%22%3A1656699023800%7D&brand=phet&ea&fuzz&memoryLimit=1000
Query: brand=phet&ea&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed: Accessibility is provided by RectangularRadioButton and RectangularRadioButtonItem.labelContent. Additional PDOM content in the provided Node could break accessibility.
Error: Assertion failed: Accessibility is provided by RectangularRadioButton and RectangularRadioButtonItem.labelContent. Additional PDOM content in the provided Node could break accessibility.
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1656695400610/assert/js/assert.js:28:13)
at assert (RectangularRadioButtonGroup.ts:289:16)
at (MixIsotopesScreenView.js:421:4)
at (MixIsotopesScreenView.js:260:43)
at (MixIsotopesScreen.js:38:15)
at createView (Screen.ts:289:22)
at initializeView (Sim.ts:841:15)
at (Sim.ts:849:10)
id: Bayes Puppeteer
Snapshot from 7/1/2022, 11:10:00 AM
```

I'll investigate.

**Comment by jbphet** [comment]
@jessegreenberg - The assertion that is causing this CT failure is coming from some code that you added in https://github.com/phetsims/sun/commit/ee394ab10393cec3dcc4f1ad532101bdb237f26b for https://github.com/phetsims/sun/issues/771.

It looks like this was added because we don't want Scenery Node instances that already have phet-io stuff being used as nodes in radio button groups, but I don't really understand why, what I should do about it, or why it's coming up for a non-instrumented sim.  In this particular case, the IAAM code (which is quite old) is using convenient common-code objects for the needed visual representations.  I've circled them in the screenshot below.  What's your guidance on how to handle this?

![image](https://user-images.githubusercontent.com/3082831/176962985-faf69ead-f3dc-43ba-862a-f2dae71ae14a.png)

**Comment by jessegreenberg** [comment]
This assertion is trying to prevent putting additional PDOM content inside the radio button through the icon can because this can create malformed HTML and a poor experience for a screen reader user. In this case, they would likely find the slider nested under the radio button.

One way to handle this is to remove that element from the PDOM by setting its tagName to `null`, see https://github.com/phetsims/isotopes-and-atomic-mass/commit/d24fc5671aa6a7e94b2f76e833c0dccd88303dd4.

**Comment by jbphet** [comment]
Thanks @jessegreenberg.  I understand the situation better, and your fix took care of the CT problem.

![image](https://user-images.githubusercontent.com/3082831/176975015-be93fc3e-978d-4a66-be80-30e88261f40c.png)

Closing.
